includes/Plugin_Context.php

```diff
@@ -85,4 +85,15 @@ public function location() {
 
 		return $path;
 	}
+
+	/**
+	 * Returns the plugin absolute path of the main file.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string The absolute path to the plugin main file.
+	 */
+	public function abspath() {
+		return $this->main_file;
+	}
```

Comment:
This method isn't necessary. You can always use `WP_PLUGIN_DIR` plus the plugin's basename (via the above `basname()` method) to get the path to the plugin main file.